<ui version="4.0" >
 <class>floatParameter</class>
 <widget class="QWidget" name="floatParameter" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>115</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="floatParameterGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Parameter</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_1" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Value:</string>
            </property>
            <property name="buddy" >
             <cstring>floatParameterValue</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="floatParameterValue" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Set the current value of this parameter between MIN and MAX.</string>
            </property>
            <property name="correctionMode" >
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="decimals" >
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="floatParameterSlider" >
          <property name="toolTip" >
           <string>Set the current value of this parameter between MIN and MAX.</string>
          </property>
          <property name="autoFillBackground" >
           <bool>false</bool>
          </property>
          <property name="maximum" >
           <number>10000</number>
          </property>
          <property name="singleStep" >
           <number>1</number>
          </property>
          <property name="pageStep" >
           <number>1000</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance" >
           <bool>false</bool>
          </property>
          <property name="tickPosition" >
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Min:</string>
            </property>
            <property name="buddy" >
             <cstring>floatParameterMin</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="floatParameterMin" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Set the minimum value for this parameter.</string>
            </property>
            <property name="correctionMode" >
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="decimals" >
             <number>10</number>
            </property>
            <property name="maximum" >
             <double>10000000.000000000000000</double>
            </property>
            <property name="minimum" >
             <double>-10000000.000000000000000</double>
            </property>
            <property name="value" >
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Max:</string>
            </property>
            <property name="buddy" >
             <cstring>floatParameterMax</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="floatParameterMax" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip" >
             <string>Set the maximum value for this parameter.</string>
            </property>
            <property name="correctionMode" >
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="decimals" >
             <number>10</number>
            </property>
            <property name="maximum" >
             <double>10000000.000000000000000</double>
            </property>
            <property name="minimum" >
             <double>-10000000.000000000000000</double>
            </property>
            <property name="value" >
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
